.github/workflows/node-gyp.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [macos-latest, ubuntu-latest, windows-latest]
-        python: ["3.8", "3.x", "3.12-dev"]
+        python: ["3.8", "3.x", "3.12"]
```

Comment:
What's `3.x` supposed to be?

Reply:
See the section on `x-ranges`.  https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#using-the-python-version-input

It basically translates to the latest current production release (currently v7.11.5).